src/test/java/org/codelibs/fess/crawler/transformer/FessFileTransformerTest.java

```diff
@@ -216,7 +220,7 @@ public void test_getSite_unexpected() {
         assertEquals(exp, transformer.getSiteOnFile(url, "UTF-8"));
 
         url = "file:///";
-        exp = "///";
```

Comment:
We need to keep this test case.
It's better not to modify components other than GitBucketDataStore.